### PR TITLE
Update docs versions

### PR DESCRIPTION
**Pull request by sam-heilbron**
# Description

Update docs versions

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/active_versions.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "latest": "1.6.0",
+  "latest": "1.6.2",
```
Comment (kdorosh) [inline docs/active_versions.json:2]: let's wait for 1.6.3 so we can get the docs search fix?

Comment (sam-heilbron) [reply docs/active_versions.json:2]: works for me!

**Review by kdorosh — APPROVED** [review]

**Pull request merged**